sdk/documentintelligence/azure-ai-documentintelligence/samples/aio/sample_analyze_addon_query_fields_async.py

```diff
@@ -0,0 +1,98 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_analyze_addon_query_fields_async.py
+
+DESCRIPTION:
+    This sample demonstrates how to extract additional fields using the
+    add-on 'QUERY_FIELDS' capability.
+
+    This sample uses Layout model to demonstrate.
+
+    Add-on capabilities accept a list of strings containing values from the `DocumentAnalysisFeature`
+    enum class. For more information, see:
+    https://learn.microsoft.com/python/api/azure-ai-formrecognizer/azure.ai.formrecognizer.analysisfeature?view=azure-python.
```

Comment:
Since you dont have this link yet you could create an aka.ms link so that you can update it later with the correct link and it'll get updated in your samples automatically when you do.